tests/testTriggers.go

```diff
@@ -155,6 +155,10 @@ const (
 	EmailReporter = "emailReporter"
 	// CoreChecker checks if any cores got generated
 	CoreChecker = "coreChecker"
+	// PoolResizeDisk resize-disk operation on storage pool
+	PoolResizeDisk = "poolResizeDisk"
+	// PoolAddDisk resize-disk operation on storage pool
```

Comment:
change it to "resize storage pool using add-disk"

Reply:
done